### DISCUSSION TITLE
How to connect innerBorder and outerBorder xsiUpperSkin/xsiLowerSkin points?

### DISCUSSION THREAD

**Discussion by GodertdeZoetenGKNFokker** [DLR-SL/CPACS#742] [Q&A] [answered]
This is a question regarding how the part inbetween the innerBorder and outerBorder of a LED or TED should be handled in case a leadingEdgeShape is defined. This definition allows to define the xsiUpperSkin and xsiLowerSkin at the innerBorder and outerBorder. Using this definition it is clear where these two points are on the innerBorder and outerBorder. However, it is not quite clear what should happen inbetween. Which of the following is correct?

1) The points on the innerBorder and outerBorder should be connected by a straight line.
2) The points on the innerBorder and outerBorder should be connected using an iso-xsi sort of line.

In the case of a TED, both these approaches yield the same result if the trailing edge inbetween these borders is a straight line. However, when there is a kink in the trailing edge, these approaches are quite different. Approach 1) could then lead to a line that could end up outside the TE
[XsiUpperSkin_issue.zip](https://github.com/DLR-SL/CPACS/files/5438083/XsiUpperSkin_issue.zip)
D planform, while approach 2) would yield a kinked line inside the planform (I have attached a sketch to illustrate the situation).

Which approach should be taken? Or should it be another approached that I didn't mention yet?

**Comment by MarAlder** [comment, marked as answer]
As far as CPACS is concerned, I fear the approach in this case is not explicitly defined. Therefore, in TiGL I think the first case was implemented (right, @rainman110?)

It would be very interesting to see an implementation of the second approach. But I cannot say what all this entails. I put on my list to bringt this into future discussions and to specify the CPACS documentation here. Thanks, Godert!
